### DISCUSSION TITLE
Scroll to first error input field in form ?

### DISCUSSION THREAD

**Discussion by ahetawal-p** [mantinedev/mantine#3096] [Q&A]
I am using `@mantine/form` for setting up a form. 
Was wondering how can I go about scrolling to the first invalid input on form submit.

**Comment by rtivital** [comment]
You can get inputs refs and scroll to any input after validation is failed

**Comment by ctretyak** [comment]
@rtivital do you have any plans to implement auto scroll to form input with error in mantine lib?

E.g. in antd scrollToFirstError param

**Reply by rtivital** [reply]
https://mantine.dev/form/validation/#focus-first-invalid-field
